tests/zkevm/utilization_summary.log

```diff
@@ -0,0 +1,8 @@
+===== UTILIZATION SUMMARY =====
+
+TEST PATTERN                                  BYTECODE                         GAS        SIZE  ITERATIONS
+----------------------------------------------------------------------------------------------------
+JUMPDEST only                      24576/24576 (100.00%)        24576/35979000 (  0.07%)     1       24576
+JUMPDEST + JUMP                    24576/24576 (100.00%)       110592/35979000 (  0.31%)     2       12288
+JUMPDEST + PUSH0 + POP + JUMP      24576/24576 (100.00%)        86016/35979000 (  0.24%)     4        6144
+PUSH0 + POP                        24576/24576 (100.00%)        61440/35979000 (  0.17%)     2       12288
```

Comment:
This seems like a new thing being added into the repo as a whole -- If so, I don't think we want it here as the intention is to have these tests look like every other test

Reply:
Right never intended to be permanent, just seeking feedback on the tests :)